### PR TITLE
Update schedule filters

### PR DESCRIPTION
**Pull request by epsy**

### CODE REVIEW

File: pybay/views.py [hunk 1]
```diff
@@ -177,10 +177,20 @@ def _day_slots(day):
...
+    ("Python at Scale", ['performantpython', 'scalablepython', 'devops']),
 ]
 
+ALLOWED_CATEGORIES = [
+    slug
+    for _, slugs in FILTER_CATEGORIES
+    for slug in slugs
+]
+
+FILTER_CATEGORIES.append(('Misc', ['other']))
```
Comment (PirosB3) [inline pybay/views.py:191]: why aren't these added above?

Comment (epsy) [reply pybay/views.py:191]: so that they are skipped in the 'ALLOWED_CATEGORIES', which is used to figure out which things go into 'other' and 'level-1'

**Review by PirosB3 — APPROVED** [review]
One small comment, but happy to deploy if it's blocking smth

File: pybay/views.py (hunk 1 above)
Comment (PirosB3) [reply pybay/views.py:191]: oh right! thanks

**Pull request merged**